velox/exec/HashProbe.cpp

```diff
@@ -1079,6 +1087,30 @@ RowVectorPtr HashProbe::getOutputInternal(bool toSpillOutput) {
       return nullptr;
     }
 
+    if (accumulatedNumOutput > 0 ||
+        (filter_ != nullptr && numOut < numOutputBeforeFilter / 10 &&
```

Comment:
How is `/ 10` decided?  Can we increase it a little bit, say `/ 2`?

Reply:
Updated, thanks